src/follower.py

```diff
@@ -73,35 +73,60 @@ def stop(self):
 
     """
     Turn the wheels towards the last recognized object.
+    Pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
+    
+    If the direction of the object is more/less than the distance then set the camera 3 steps 
+    to the left or right. Otherwise reset the camera to 90 degrees or 0 steps. 
+    """
+    def pan_camera(self):
+        if self._turn_angle < self._distance:
+            self.turn_camera_left(3)
+        elif self._turn_angle > self._distance:
+            self.turn_camera_right(3)
+        else:
+            self.reset_camera() 
+
+
+    """
+    Definitions for turning the camera left and right.
+    Parameter: Steps 
+    """
+    def turn_camera_left(self, step):
+        self._camera.turn_left(step)
+    
+    def turn_camera_right(self, step):
+        self._camera.turn_right(step)
+
 
+    """
+    Converts the camera's angle scale to the same scale as the wheels.
```

Comment:
Remove this line. It's the same as what the next paragraph explains. It also has nothing to do with your pull request, so don't touch it.